traitsui/qt4/ui_base.py

```diff
@@ -128,16 +128,10 @@ def __init__(self, ui, parent):
         self.setLayout(layout)
 
         # Set the dialog's window flags and properties.
-        if ui.view.resizable:
-            flags = QtCore.Qt.Window
-        else:
-            flags = QtCore.Qt.Dialog | QtCore.Qt.WindowSystemMenuHint
+        flags = QtCore.Qt.Dialog
+        if not ui.view.resizable:
+            flags |= QtCore.Qt.WindowSystemMenuHint
             layout.setSizeConstraint(QtGui.QLayout.SetFixedSize)
-            if ui.view.resizable:
```

Comment:
Note: This removal is actually a drive-by. This branch is never reached because it is in the else branch where the condition is false.

Reply:
🤣  yeah i was wondering why that branch existed

Reply:
i just dug through history to find a reason for why this exists and i can see the commit where this was added but i don't see a PR associated with the commit/change - https://github.com/enthought/traitsui/commit/b61ab16c153375103923ecf0d72f15f15419f51e

Reply:
🤷